app/helpers/requests_helper.rb

```diff
@@ -100,8 +100,9 @@ def format_date(date)
     Time.zone.parse(date.to_s).strftime('%Y-%m-%d %I:%M%P')
   end
 
-  def searchworks_link(item_id, item_title)
-    link_to item_title, "#{Settings.searchworks_link}/#{item_id}", 'data-behavior': 'truncate'
+  def searchworks_link(item_id, item_title, html_options = {})
+    html_options['data-behavior'] = 'truncate'
```

Comment:
since it seems like we want that as the default behavior, what about making it the default in the absence of the caller setting it explicitly, that way we don't have to add it to either of the places that currently call `searchworks_link`.  i.e.:
`html_options['data-behavior'] = 'truncate' unless html_options['data-behavior']`